js/localization/globalize/date.js

```diff
@@ -114,8 +114,8 @@ if(Globalize && Globalize.formatDate) {
             return Globalize.locale().main('dates/calendars/gregorian/' + path);
         },
 
-        getPeriodNames: function() {
-            const json = Globalize.locale().main('dates/calendars/gregorian/dayPeriods/stand-alone/wide');
+        getPeriodNames: function(format) {
```

Comment:
```suggestion
        getPeriodNames: function(format = 'wide') {
```

Reply:
maybe it's better to have insurance from getPeriodNames(null) calling?